manifest/openshift4/coreos/ncp-openshift4.yaml

```diff
@@ -224,6 +224,16 @@ rules:
 
 
 
+
+ - apiGroups:
```

Comment:
Do we have the patch in NCP manifest template as well, usually, we want to generate all platform manifests and configmap from template?

Reply:
Yes, we generated the whole file when building sb/ob. I'm not sure if want to update the whole file for this change? I thought we will do batch update whenever a version is released?

Reply:
Yes, We will do a bath update of the whole manifest file once release branch is cut.